app/scripts/templates/settings/client_disconnect.mustache

```diff
@@ -9,20 +9,18 @@
       </p>
 
       <form novalidate>
-
-        <div class="select-row-wrapper">
-        <label class="label-helper"></label>
-        <div class="select-row">
-          <select class="disconnect-reasons">
-            <option value="reason">{{#t}}Reason for disconnecting…{{/t}}</option>
-            <option value="suspicious">{{#t}}Suspicious device{{/t}}</option>
-            <option value="lost">{{#t}}Lost or stolen device{{/t}}</option>
-            <option value="old">{{#t}}Old or replaced device{{/t}}</option>
-            <option value="duplicate">{{#t}}Duplicate device{{/t}}</option>
-            <option value="no">{{#t}}Rather not say{{/t}}</option>
-          </select>
-        </div>
-        </div>
+          <p class="disconnect-reasons fxa-radio">
+            <input id="suspicious" type="radio" name="disconnect-reasons" value="suspicious">
+              <label>{{#t}}Suspicious device{{/t}}</label><br/>
```

Comment:
![label](https://cloud.githubusercontent.com/assets/128755/21619800/50064766-d1bf-11e6-8f87-6354a2a4977f.gif)

you would need to do something like `<label> <input id="suspicious" type="radio" name="disconnect-reasons" value="suspicious"></label>` for this to make sure labels work when you click on them.